features/dd-sdk-android-webview/api/apiSurface

```diff
@@ -1,5 +1,5 @@
 object com.datadog.android.webview.WebViewTracking
   fun enable(android.webkit.WebView, List<String>, Float = 100f, com.datadog.android.api.SdkCore = Datadog.getInstance())
   class _InternalWebViewProxy
-    constructor(com.datadog.android.api.SdkCore)
```

Comment:
even though it is for the internal usage, we need to check that both Flutter and RN:

* are able to provide `WebView` instance
* are not using dynamic versioning for native SDK dependencies (otherwise existing releases of RN/Flutter will be broken)

If there is a blocker for that change, we need to have 2 constructors then: with and without `WebView`.

Reply:
yes...but for SR to work we need that unique id for the WebView, this is a very important part in the logic. I need to discuss with @louiszawadzki and @fuzzybinary about this.

Reply:
Essentially, since the only info you need is a Webview identifier, and not the webview itself, you could delegate that and allow a null value for the Cross Platform SDKs, and maybe not init the SessionReplay consumer if this value is missing.

Reply:
yeah...I might have to do something to support a null there